packages/gafl-webapp-service/src/pages/licence-details/licence-length/__tests__/result-function.spec.js

```diff
@@ -0,0 +1,54 @@
+import resultFunction from '../result-function'
+import { CommonResults } from '../../../../constants.js'
+import { licenceToStart } from '../../licence-to-start/update-transaction.js'
+
+describe('licence-length > result-function', () => {
+  const mockStatusCacheGet = jest.fn(() => ({}))
+  const mockTransactionCacheGet = jest.fn(() => ({}))
```

Comment:
This has always been a grotty pattern, please could you incorporate into the mockRequest?